src/main/cpp/src/config/json_config.cc

```diff
@@ -779,4 +784,20 @@ void GenomicsDBImportConfig::read_from_file(const std::string& filename, const i
   m_no_mandatory_VCF_fields = false;
   if (json_doc.HasMember("no_mandatory_VCF_fields") && json_doc["no_mandatory_VCF_fields"].IsBool())
     m_no_mandatory_VCF_fields = json_doc["no_mandatory_VCF_fields"].GetBool();
+
```

Comment:
Did you mean to use the set_config_field function for these as well?

Reply:
Yes, I did, just wanted to keep changes directly related to shared posix filesystem optimizations for this PR.

Reply:
Good catch. Yes, that was an oversight.